website/www/site/content/en/documentation/runners/direct.md

```diff
@@ -0,0 +1,165 @@
+---
+type: runners
```

Comment:
is that `type` necessary?

Reply:
This syntax offers the same feature as `layout: runners` in Jekyll. The Runners page is inside documentation, so I use it to separate Runners layout with Documentation layout. 🙂